### PR TITLE
Add benchmarking for mobilenetv2

### PR DESCRIPTION
**Pull request by alrevuelta**
Add benchmarking for mobilenetv2

### CODE REVIEW

**Comment by nopeslide** [comment]
could you remove the `.DS_Store` files?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ build
 profiling
 benchmarks
 dump.txt
+.DS_Store
```
Comment (nopeslide) [inline .gitignore:5]:
```suggestion
**/.DS_Store
```

Comment (alrevuelta) [reply .gitignore:5]: Looks like works without it, but ok :)

Comment (nopeslide) [reply .gitignore]: ok. was not sure of the default behaviour of .gitignore here

**Comment by nopeslide** [comment]
could you squash the the `.DS_Store` related commits so it does not clutter our history?

**Pull request merged**